crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -419,6 +406,15 @@ pub struct SuiSystemStateInnerV1 {
     // TODO: Use getters instead of all pub.
 }
 
+impl SuiSystemStateInnerV1 {
+    pub fn new_for_testing(epoch: EpochId) -> Self {
```

Comment:
`#[cfg(test)]` ?

Reply:
These test cfgs generally don't work because they are not carried out of the crate.

Reply:
will feature gates works?

Reply:
I don't see a way to make it work though